pkg/controller/dnsrecord/actuator.go

```diff
@@ -168,15 +156,12 @@ func (a *actuator) getDomainName(ctx context.Context, dns *extensionsv1alpha1.DN
 		// getting all domain names of the account and searching for the longest domain name that is a suffix of dns.spec.Name
 		domainNames, err := dnsClient.GetDomainNames(ctx)
 		if err != nil {
-			return "", &reconcilerutils.RequeueAfterError{
-				Cause:        fmt.Errorf("could not get DNS domain names: %+v", err),
-				RequeueAfter: requeueAfterOnProviderError,
-			}
+			return "", fmt.Errorf("could not get DNS domain names: %+v", err)
 		}
 		a.logger.Info("Got DNS domain names", "domainNames", domainNames, "dnsrecord", kutil.ObjectName(dns))
 		domainName := dnsrecord.FindZoneForName(domainNames, dns.Spec.Name)
 		if domainName == "" {
-			return "", fmt.Errorf("could not find DNS domain name for name %s", dns.Spec.Name)
+			return "", wrapAliClientError(err, fmt.Sprintf("could not find DNS domain name for name %s", dns.Spec.Name))
```

Comment:
why need to wrapAliClientError?

Reply:
In order to re-enqueue the event if error is caused by Throttling

Reply:
`err` is alway nil. No wrap is needed.